settings/PandoraSettings_EDepReco.xml

```diff
@@ -48,19 +46,25 @@
         <ShouldRunStitching>true</ShouldRunStitching>
         <ShouldRunCosmicHitRemoval>false</ShouldRunCosmicHitRemoval>
         <ShouldRunSlicing>false</ShouldRunSlicing>
-        <ShouldRunNeutrinoRecoOption>false</ShouldRunNeutrinoRecoOption>
+        <ShouldRunNeutrinoRecoOption>true</ShouldRunNeutrinoRecoOption>
```

Comment:
I think all these `ShouldRun...` options can/should now be safely removed from the EDepReco.xml file, as these are the details provided by (and I think overridden by) the new `-r RecoOption` command line flag.